.travis.yml

```diff
@@ -11,19 +11,19 @@ services:
   - docker
 env:
   global:
-    - REPO_NAME=pyoptsparse
+    - REPO_NAME=$(eval basename $TRAVIS_REPO_SLUG)
```

Comment:
Just a curiosity here, how does this command work? what kind of env variable is `$TRAVIS_REPO_SLUG`?

Reply:
It's set to `mdolab/pyoptsparse` for this repo. `basename` picks out the last portion after all the `/` so it's a convenient way to get the repo name programmatically.